sass/main.scss

```diff
@@ -940,6 +938,57 @@ footer {
         grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
+
+}
+
+@media screen and (min-width: 1700px) {
+    nav {
+        font-size: 1.5em;
+
+        #login {
+	    padding: 15px 30px;
+	}
+
+	#logo {
+	    width: 120px;
+	    height: 120px;
+	}
+
+	#mlh-badge{
+	    width: 140px;
+	}
+
```

Comment:
nit: extra newline here between closing braces should be removed

Reply:
[fixed in this commit](https://github.com/codeRIT/brickhack.io/pull/1053/commits/c36383cac36d0d0da2f36bb0f49511309066d7a6)